run.py

```diff
@@ -452,8 +452,9 @@ def fetch_test_details(var) -> dict:
                 res.append(tmp)
             total_time = "0s"
             send_to_cephci = post_results or post_to_report_portal
-            email_results(res, run_id, trigger_user, run_dir, total_time, send_to_cephci)
-            return
+            info = f"Exception hit during node creation.Traceback: {traceback.format_exc(limit=2)}"
```

Comment:
Can we keep this `environment setup` status attribute in html page and need `success` or `OKAY` for successful environment setup.

Reply:
if it is node creation failure, it would be easy to address as environment failure. I think it may be not that easy with trace-back failure to say whether its environment or test failure straightaway.

Reply:
yeah, we can't have traceback information in the emails. just add the status. use proper flag.

Reply:
Made the suggested changes.